scripts/build/electron-create-appdir.sh

```diff
@@ -105,65 +88,45 @@ Icon=$APPDIR_ICON_FILENAME
 Type=Application
 EOF
 
-cp "$ARGV_ICON" "$APPDIR_PATH/$APPDIR_ICON_FILENAME.png"
-cp -rf "$ARGV_PACKAGE"/* "$APPDIR_PATH/usr/bin"
+cp "$ARGV_ICON" "$ARGV_OUTPUT/$APPDIR_ICON_FILENAME.png"
+mkdir -p "$ARGV_OUTPUT/usr/bin"
+cp -rf "$ARGV_PACKAGE"/* "$ARGV_OUTPUT/usr/bin"
 
 # Compress binaries
-upx -9 "$APPDIR_PATH/usr/bin/$ARGV_BINARY"
+upx -9 "$ARGV_OUTPUT/usr/bin/$ARGV_BINARY"
 
 # upx fails with an error if .so are not executables
-chmod +x "$APPDIR_PATH"/usr/bin/*.so*
+chmod +x "$ARGV_OUTPUT"/usr/bin/*.so*
 
 # UPX fails for some reason with some other so libraries
 # other than libnode.so in the x86 build
 if [ "$ARGV_ARCHITECTURE" == "x86" ]; then
-  upx -9 "$APPDIR_PATH"/usr/bin/libnode.so
+  upx -9 "$ARGV_OUTPUT"/usr/bin/libnode.so
 
 else
-  upx -9 "$APPDIR_PATH"/usr/bin/*.so*
+  upx -9 "$ARGV_OUTPUT"/usr/bin/*.so*
 fi
 
-# Generate AppImage
-mkdir -p "$(dirname "$ARGV_OUTPUT")"
-rm -f "$ARGV_OUTPUT"
-
 APPIMAGES_TAG=6
-APPIMAGES_GITHUB_RELEASE_BASE_URL=https://github.com/probonopd/AppImageKit/releases/download/$APPIMAGES_TAG
-APPIMAGEASSISTANT_PATH=$TMPDIR/AppImageAssistant.AppImage
 
 ./scripts/build/download-tool.sh -x \
   -u "https://raw.githubusercontent.com/probonopd/AppImageKit/$APPIMAGES_TAG/desktopintegration" \
```

Comment:
To better match the command later in this script, I wonder if it'd be worth pulling out:
```bash
APPIMAGES_GITHUB_RAW_BASE_URL=https://raw.githubusercontent.com/probonopd/AppImageKit
```